packages/dialtone-vue2/components/image_viewer/image_viewer.vue

```diff
@@ -82,6 +85,15 @@ export default {
   mixins: [Modal],
 
   props: {
+    /**
+     * By default the portal appends to the body. We can modify
+     * this behaviour by passing an appendTo prop that points to an id.
+    */
+    appendTo: {
+      type: String,
+      default: 'body',
+    },
+
```

Comment:
Please update this comment to be more specific about what appendTo prop expects here.

Reply:
umm would this be ok?
```
/**
 * By default the portal appends to the body of the root parent. We can modify
 * this behaviour by passing an appendTo prop that points to an id or an html tag from the root of the parent.
 * type: string, default: 'body'
*/
```

Reply:
Add something like this to the comment: The appendTo prop expects a CSS selector string or an actual DOM node.

Got it from: https://vuejs.org/guide/built-ins/teleport

Reply:
The allowed types can be 'String' or 'HTMLElement'